ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -409,10 +408,9 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         if (this.isMode(WorkspaceEditMode.Duplicate)) {
           this.setState({workspace: {
             ...this.props.workspace,
-            // These are the only fields which are not automatically handled/differentiated
+            // This is the only field which is not automatically handled/differentiated
             // on the API level.
-            name: 'Duplicate of ' + this.props.workspace.name,
-            namespace: userProfileStore.getValue().profile.freeTierBillingProjectName
+            name: 'Duplicate of ' + this.props.workspace.name
```

Comment:
I don't think we want to do this. I guess the "duplicate of" stuff was there before, but I think you wan to kill this property completely and fixup any places using the duplicated one.